src/components/TeamGeneratorPage/ImportPlayerList.js

```diff
@@ -12,6 +12,12 @@ const ImportPlayerList = ({ callback }) => {
 		callback(list);
 	}
 
+	useEffect(() => {
+		const playersList = getStoredPlayersList();
+		setPlayerList(playerList);
+	}, []);
+	window.addEventListener('clear', () => setPlayerList([]));
```

Comment:
Why do you need this?

Reply:
to manage states and logic from different components